packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -1356,6 +1360,81 @@ describe("IModelTransformer", () => {
     targetIModelDb.close();
   });
 
+  it("should log unresolved references", async () => {
+    const iModelShared: SnapshotDb =
+      IModelTransformerTestUtils.createSharedIModel(outputDir, ["A", "B"]);
+    const iModelA: SnapshotDb = IModelTransformerTestUtils.createTeamIModel(
+      outputDir,
+      "A",
+      Point3d.create(0, 0, 0),
+      ColorDef.green
+    );
+    IModelTransformerTestUtils.assertTeamIModelContents(iModelA, "A");
+    const iModelExporterA = new IModelExporter(iModelA);
+
+    // Exclude element
+    const excludedId = iModelA.elements.queryElementIdByCode(
+      Subject.createCode(iModelA, IModel.rootSubjectId, "Context")
+    );
+    assert.isDefined(excludedId);
+    iModelExporterA.excludeElement(excludedId!);
+
+    const subjectId: Id64String = IModelTransformerTestUtils.querySubjectId(
+      iModelShared,
+      "A"
+    );
+    const transformerA2S = new IModelTransformer(
+      iModelExporterA,
+      iModelShared,
+      { targetScopeElementId: subjectId, danglingReferencesBehavior: "ignore" }
```

Comment:
Note: I had to add danglingreferncesbehavior ignore or this test errored out.

Reply:
figured it out, LGTM